src/Particular.CodeRules/Particular.ruleset

```diff
@@ -230,4 +230,54 @@
     <Rule Id="CA5403" Action="None" />             <!-- Do not hard-code certificate -->
     <Rule Id="CA9999" Action="None" />             <!-- Analyzer version mismatch -->
   </Rules>
-</RuleSet>
\ No newline at end of file
+  <Rules AnalyzerId="StyleCop.Analyzers" RuleNamespace="StyleCop.Analyzers">
+    <!-- rules we may want to enforce eventually -->
+    <Rule Id="SA0001" Action="None" />             <!-- XML comment analysis disabled -->
+    <Rule Id="SA1005" Action="None" />             <!-- Single line comments should begin with single space -->
+    <Rule Id="SA1009" Action="None" />             <!-- Closing parenthesis should be spaced correctly -->
+    <Rule Id="SA1012" Action="None" />             <!-- Opening braces should be spaced correctly -->
+    <Rule Id="SA1013" Action="None" />             <!-- Closing braces should be spaced correctly -->
+    <Rule Id="SA1028" Action="None" />             <!-- Code should not contain trailing whitespace -->
+    <Rule Id="SA1121" Action="None" />             <!-- Use built-in type alias -->
```

Comment:
Seems to be a low hanging fruit and general a good recomendation no?

Reply:
Again, just too many violations, so we put it on the "later" list.

Reply:
Isn't this a finite set of Search and Replace that can be pulled of in a few minutes?

Reply:
Maybe we can re-evaluate. Keep in mind, there are several dozen repos to review and we already have a lengthy list of ones that we've committed to. I believe a good number of these could be considered low-hanging fruit. Others could probably be done with some ReSharper and/or VS formatting rules.

Reply:
BTW I'm merely thinking out loud and ask requested provide opinions. Totally OK if my opinion is ignored 😉

Reply:
@danielmarbach we reviewed this one and we agree, we're going to fix it.